assets/app/actions/dispatchActions.js

```diff
@@ -0,0 +1,72 @@
+import { dispatch } from '../store';
```

Comment:
I have a demo design app put together for a different client that's developed opinions about these very ideas here: https://github.com/18F/react-redux-event-demo